Wireframe/index.html

```diff
@@ -4,10 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+    <header>
+      <h1>Git Made Easy</h1>
+      <h2>An explanation of the mysteries of Git.</h2>
+    </header>
+
+    <div class="main">
+      <article class="hero">
+        <img>
+        <h2>What is Git?</h2>
+        <p>Git is the backbone of modern software development, a powerful version control system that helps developers track changes, collaborate, and maintain project history. Whether you're working solo or in a large team, Git keeps your code organized and recoverable. Want to know how it works and why it's so essential? Click to dive in!
+        </p>
+        <a target="_blank" href="https://www.freecodecamp.org/news/what-is-git-and-how-to-use-it-c341b049ae61/"><button>READ MORE</button></a>
+       </article>
+    </div>
+
+    <div class="content">
+      <article class="card">
+        <img>
+        <h2>Why do developers need Git?</h2>
+        <p>In today's fast-paced development world, Git has become an indispensable tool for managing projects. From version control to collaboration, it enables teams to work efficiently without stepping on each other's toes. Curious about how Git makes coding smoother and safer for developers? Learn more here.
+        </p>
+        <a target="_blank" href="https://www.atlassian.com/git/tutorials/why-git"><button>READ MORE</button></a>
+      </article>
+
+      <article class="card">
+        <img>
```

Comment:
Is this valid HTML? What [attributes] does this img tag need to make it show images and to make it describe the image it is showing?